src/main/java/com/lesson9/Bandlist/mapper/BandMapper.java

```diff
@@ -1,30 +1,38 @@
 package com.lesson9.Bandlist.mapper;
 
 import com.lesson9.Bandlist.entity.Band;
+import org.apache.ibatis.annotations.Delete;
 import org.apache.ibatis.annotations.Insert;
 import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Options;
 import org.apache.ibatis.annotations.Select;
+import org.apache.ibatis.annotations.Update;
 
 import java.time.ZonedDateTime;
 import java.util.List;
 
 @Mapper
 public interface BandMapper {
-    @Select("SELECT DISTINCT b.id, b.band_name, b.act_Announcement_Date FROM bands b INNER JOIN members m ON b.id = m.band_id")
+
+    @Select("SELECT DISTINCT b.id, b.band_name, b.act_announcement_date FROM bands b LEFT JOIN members m ON b.id = m.band_id")
     List<Band> findAllUniqueBands();
 
     @Select("SELECT * FROM bands WHERE id = #{id}")
     Band findById(int id);
```

Comment:
ここでOptionalで返してしまいましょう。
```suggestion
   Optional<Band> findById(int id);
```